packages/vincent-tools/vincent-tool-sdk/src/lib/playground.ts

```diff
@@ -0,0 +1,342 @@
+import z from 'zod';
+import { validateVincentPolicyDef } from './vincent-policy';
+import { validateVincentToolDef } from './vincent-tool';
+
+// Define your tool schema
+const myToolSchema = z.object({
+  action: z.string(),
+  target: z.string(),
+  amount: z.number(),
+});
+
+// Define policy schemas
+const policy1Schema = z.object({
+  targetAllowed: z.boolean(),
+  actionType: z.string(),
+});
+const userParams1Schema = z.object({ allowedTargets: z.array(z.string()) });
+const commitParams1Schema = z.object({ confirmation: z.boolean() });
+
+// Define the result schemas for policy1
+const policy1EvalAllowResult = z.string(); // Result when allowed
+const policy1EvalDenyResult = z.object({ reason: z.string() }); // Result when denied
+const policy1CommitAllowResult = z.object({ success: z.boolean() });
+const policy1CommitDenyResult = z.object({ errorCode: z.number() });
```

Comment:
Would it make sense to add this playground to the readme or docs? Seems like something app devs will want to do

Also, can we have more descriptive names? What does `policy1` do and what it has different from `policy2` and `policy3`?